src/sub-blocks/BasicCard/BasicCard.scss

```diff
@@ -4,15 +4,42 @@
 $block: '.#{$ns}basic-card';
 
 #{$block} {
+    $iconSizeL: 32px;
```

Comment:
Why it's name is size L? 
If the size depends on position maybe we can name it something like `iconSizePositionLeft`?

Reply:
Good point